Packs/AzureStorage/Integrations/AzureStorage/AzureStorage.py

```diff
@@ -47,54 +47,57 @@ def __init__(self, app_id: str, subscription_id: str, resource_group_name: str,
         self.connection_type = connection_type
 
     @logger
-    def storage_account_list_request(self, account_name: str) -> Dict:
+    def storage_account_list_request(self, account_name: str, resource_group_name: str, subscription_id: str) -> Dict:
         """
             Send the get storage account/s request to the API.
         Args:
             account_name: The storage account name, optional.
+            resource_group_name: The resource group name.
+            subscription_id: The subscription id.
 
         Returns:
             The json response from the API call.
         """
 
         return self.ms_client.http_request(
             method='GET',
-            url_suffix=f'/resourceGroups/{self.resource_group_name}/providers/Microsoft.Storage/storageAccounts/'
-                       f'{account_name}',
-            params={
-                'api-version': API_VERSION,
-            }
+            full_url=f'https://management.azure.com/subscriptions/{subscription_id}/resourceGroups/{resource_group_name}/providers/Microsoft.Storage/\
```

Comment:
Here as well - the `https://management.azure.com/subscriptions` prefix could be a const.

Reply:
Done

Reply:
please use the const in line 354.

Reply:
and in AzureKubernetes line 132.

Reply:
and in azur netwrork sec groups line 123.

Reply:
Usually would be better to contact paths with URL.path.join.

Reply:
next time:)